src/devices/honyar.ts

```diff
@@ -128,6 +129,38 @@ const definitions: Definition[] = [
             await reporting.onOff(endpoint3);
         },
     },
+    {
+        zigbeeModel: ['000a0abb\u0000', 'RH5000_SmartOutlet'],
+        model: 'IHC8223AL',
+        vendor: 'Honyar',
+        description: 'Smart Power Socket 10A (with power monitoring)',
+        extend: [
+            onOff({powerOnBehavior: false}),
+            electricityMeter({cluster: 'honyar'}),
+        ],
+        onEvent: async (type, data, device) => {
```

Comment:
Can you try without this? I hope this device support reporting so polling is not needed.

Reply:
I've already tried. Proactively report a problem, you have to proactively poll.

Reply:
Can you try reusing existing polling code?

Try with:

```suggestion
        options: [exposes.options.measurement_poll_interval()],
        onEvent: (type, data, device, options) => tuya.onEventMeasurementPoll(type, data, device, options),
```